lib/forever/cli.js

```diff
@@ -196,7 +198,32 @@ function checkColumn(name) {
 // `forever.startDaemon`
 //
 var getOptions = cli.getOptions = function (file) {
-  var options = {};
+  var options = {},
+      absFile = path.isAbsolute(file) ? file : path.resolve(process.cwd(), file),
+      configKeys = [
+        'pidFile', 'logFile', 'errFile', 'watch', 'minUptime', 'append',
+        'silent', 'outFile', 'max', 'command', 'path', 'spinSleepTime',
+        'sourceDir', 'workingDir', 'uid', 'watchDirectory', 'watchIgnore',
+        'killTree', 'killSignal', 'id'
+      ],
+      jsonConfig = {};
+
+  //
+  // Load JSON configuration values
+  //
+  if (path.extname(file) === '.json') {
+    jsonConfig = path.isAbsolute(file) ? shush(file) : shush(absFile);
+    for (var k in jsonConfig) {
```

Comment:
Sorry I am being super picky here but just because we got rid of lodash doesn't mean we have to throw functional programming out the window :smile: 

``` js
jsonConfig = Object.keys(jsonConfig)
  .reduce(function (acc, k) {
    if (!~configKeys.indexOf(k) && k !== 'script') { acc[k] = jsonConfig[k]; }
    return acc;
  }, {});
```